.github/workflows/discord-build-notifier.yml

```diff
@@ -26,55 +34,86 @@ jobs:
     name: Notify Discord
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
+      - name: Checkout repository
+        uses: actions/checkout@v4
         with:
           fetch-depth: 1000
           fetch-tags: true
-        if: inputs.build_number == '???'
-
-      - name: Get git commit description
-        # language=bash
-        run: |
-          echo "GIT_COMMIT_DESC=$(git describe --tags --long)" >> $GITHUB_ENV
-          echo "GIT_COMMIT_TAG=$(git describe --tags --abbrev=0)" >> $GITHUB_ENV
-        if: inputs.build_number == '???'
+        if: inputs.build_number == ''
```

Comment:
Do we have anything that uses a build_number? No idea what this is.
But it all looks good we'll see if things break.

Reply:
If a build_number cannot be set, the bash script will set the value to `???`. I've run `set +e` in the bash script to allow for command substitution failures, and this is because I've enabled logging in case something goes wrong. I can add more logging and docs before we pull tomorrow.

Reply:
also yes the discord action uses the build number